src/index.css

```
@@ -17,7 +17,22 @@ code {
 }
 
 .p-button:hover {
-  transform: scale(1.02);
+  transform: scale(1.015);
+}
+
+.twitter:hover {
+  background-color: #1DA1F2 !important;
+  color: #fff !important;
+}
+
+.youtube:hover {
+  background-color: #FF0000 !important;
+  color: #fff !important;
+}
+
+.github:hover {
+  background-color: #211F1F !important;
+  color: #fff !important;
 }
```

are these `!important` necessary? In my opinion we should try to avoid using it unless it's absolutely necessary. In this case it's a simple case of **specificity** (you can check [these docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception) to know more).
How about we do this:
```suggestion
.p-button.p-button-outlined:not(a):not(.p-disabled).twitter:hover {
  background-color: #1DA1F2;
  color: #fff !important;
}

.p-button.p-button-outlined:not(a):not(.p-disabled).youtube:hover {
  background-color: #FF0000;
  color: #fff !important;
}

.p-button.p-button-outlined:not(a):not(.p-disabled).github:hover {
  background-color: #211F1F;
  color: #fff !important;
}
```

I didn't remove the `color` property `!important` because we are using inline styles... which I think should be refactored too 😅, but changes should be made incrementally and not all in one go 👍  
@Kunal0007 what do you think 🙂?

---

Thank you for your suggestions, great work 👍. Yes, changes should be made incrementally. Someone other can make an issue or a PR for these changes it will increase contributions 🙂.

---

I prefer to not have `!important` if possible too, these sorts of improvements won't happen later

---

@eddiejaoude made some changes